docs/ServerAPIs.md

```diff
@@ -20,13 +32,81 @@ A JSON object indicating the server's health status. For example:
 
 The `status` field currently only returns the value `"healthy"`.
 
-### Possible HTTP Status Codes
+## Query Pre-loaded Vector Database (POST `/query-vector`)
+
+### Description
+
+This endpoint allows you to query the vector database used in the application. It's primarily intended for testing
+and verification purposes, not for client-side use. The endpoint accepts a query string and returns the top 4 matches
+from the database.
+
+### Pre-requisites
+
+Before using this endpoint, ensure you've completed the following one-time setup:
```

Comment:
Great catch, @klown.

Having `enableRag` to `true` by default will force ppl to setup the vector store, which is unnecessary. It might be easier to use to have this flag turned off to start with. I've changed it to `false` as the default value.

The setup steps for RAG has been moved to the main README which is referenced in the `ServerAPIs.md`.